### DISCUSSION TITLE
Bromite is NOT being installed from f-droid

### DISCUSSION THREAD

**Discussion by oldphones68** [bromite/bromite#1602] [Q&A]
Checked on both Android 11 & 6. (Samsung Galaxy M51 & Huawei Y6ii respectively).

When trying to install, it says 'not installed' after few seconds.

Wanted to enter into the Issue tracker but that is much clumsy.

**Comment by Huy-Ngo** [comment]
I have the same problem. I'm on Xiaomi, so I don't think the hardware is relevant. Might be some problem with Bromite's f-droid server.

**Reply by uazo** [reply]
> Might be some problem with Bromite's f-droid server

just tried it on a Mi5 (arm64) LineageOS Android 11, installs perfectly with f-droid

**Reply by Huy-Ngo** [reply]
It seems to be inconsistent but the chance it can download for me is nearly null

**Reply by csagan5** [reply]
The downloads are hosted on GitHub, you should check the F-Droid client logs to see what is failing.

**Comment by oldphones68** [comment]
Yesterday, I could update it finally via f-Droid.
@uazo @Huy-Ngo
